frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/sc-node-list.tsx

```diff
@@ -17,11 +18,16 @@ import '../ocs-install.scss';
 
 const getRows: GetRows = ({ componentProps, customData }) => {
   const { data } = componentProps;
-  const { nodes, setNodes, filteredNodes } = customData;
+  const { nodes, setNodes, filteredNodes, isHostNameSame } = customData;
+  let filteredData: NodeKind[] = [];
 
   const nodeList = filteredNodes ?? data.map(getName);
```

Comment:
Instead of setting nodeList -> data.map(getName) in case of no `filteredNodes` , You want to directly set `filteredData` to the `data.map(getName)` and avoid redundant filtering below.